app/services/group_stage_service.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class GroupStageService
+  def self.generate_group_stage(groups)
+    return false if groups.length.zero?
+
+    average_group_size = (groups.flatten.length.to_f / groups.length.to_f)
+    if (average_group_size % 1).zero?
```

Comment:
not really necesasry to assign this to a var, but if you think it improvdes readability, i'm fine with that...

Reply:
I do :+1: